db/migrate/20220705235932_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Great job creating your data model! However, the posts table is missing the counters, it would be great if you added them:

![Captura de pantalla 2022-07-05 a la(s) 8 26 24 p  m](https://user-images.githubusercontent.com/83717285/177447585-61814e05-4cea-4c45-928a-58d99f7a1bf3.png)